lib/client/getStats.ts

```diff
@@ -0,0 +1,298 @@
+import bent from 'bent'
+import BigNumber from 'bignumber.js'
+import Web3 from 'web3'
+import { always } from 'ramda'
+import { addresses } from '../addresses'
+import { createDevkitContract, DevkitContract } from '../contract'
+
+const DEV_OWN_HTTP_PROVIDER = 'https://devprotocolnode.net/ethereum/mainnet'
+const DEV_GRAPHQL_ENDPOINT = 'https://api.devprotocol.xyz/v1/graphql'
+const THEGRAPH_UNISWAP_ENDPOINT =
+	'https://api.thegraph.com/subgraphs/name/uniswap/uniswap-v2'
+const DEV_TEAM_WALLET_ADDRESS = '0xe23fe51187a807d56189212591f5525127003bdf'
+const DEV_CONTRACT_ADDRESS = '0x5caf454ba92e6f2c929df14667ee360ed9fd5b26'
+
+const falsyOrZero = <T>(num?: T): T | 0 => (num ? num : 0)
+const toNaturalBasis = new BigNumber(10).pow(18)
+export const toNaturalNumber = (num?: number | string | BigNumber): BigNumber =>
+	new BigNumber(falsyOrZero(num)).div(toNaturalBasis)
+
+export type DevStats = {
+	readonly devPrice: string
+	readonly totalCap: string
+	readonly marketCap: string
+	readonly stakingRatio: string
+	readonly stakingAmount: string
+	readonly stakerAPY: string
+	readonly creatorAPY: string
+	readonly annualSupplyGrowthRatio: string
+	readonly creatorsRewardsDEV: string
+	readonly creatorsRewardsUSD: string
+}
+
+type graphBundle = {
+	readonly data: {
+		readonly bundle: {
+			readonly ethPrice: string
+		}
+	}
+}
+
+type graphToken = {
+	readonly data: {
+		readonly token: {
+			readonly derivedETH: string
+		}
+	}
+}
+
+type propertyFactoryCreateAggregate = {
+	readonly property_factory_create_aggregate: {
+		readonly aggregate: {
+			readonly count: string
+		}
+	}
+}
+
+export type GetStatsCaller = () => Promise<DevStats>
+type GetEthPriceCaller = () => Promise<graphBundle>
+type GetDevEthPriceCaller = () => Promise<graphToken>
+
+// eslint-disable-next-line functional/functional-parameters
+const getEthPrice: GetEthPriceCaller = () => {
+	return bent(
+		THEGRAPH_UNISWAP_ENDPOINT,
+		'POST',
+		'json'
+	)('', {
+		query: `{ bundle(id: 1) { id ethPrice } }`,
+		variables: null,
+	}).then((r) => r as graphBundle)
+}
+
+// eslint-disable-next-line functional/functional-parameters
```

Comment:
ESLint's `functional/functional-parameters` rule can be removed by using Ramda's `always`. Like the following:

```ts
const getEthPrice: GetEthPriceCaller = always(bent(
	THEGRAPH_UNISWAP_ENDPOINT,
	'POST',
	'json'
)('', {
	query: `{ bundle(id: 1) { id ethPrice } }`,
	variables: null,
}).then((r) => r as graphBundle))
```